lib/handlebars_helpers.js

```diff
@@ -0,0 +1,7 @@
+var Handlebars = require('handlebars');
+
+Handlebars.registerHelper('address_string', function(object) {
+  var addressArr = [object.city, object.region, object.countryCode];
+  addressArr = addressArr.filter(function(elem) {return elem != undefined});
+  return addressArr.join(', ');
```

Comment:
This is neat! :+1: 

Small suggestion, since we use underscorejs we can rewrite it using `_.compact` method

```javascript
var _ = require('underscore');
var Handlebars = require('handlebars');

Handlebars.registerHelper('address_string', function(object) {
  var addressArr = [object.city, object.region, object.countryCode];
  return _.compact(addressArr).join(', ');
```